PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace PanoptesNetClient
+{
+    public class ApiClient
+    {
+        static HttpClient client = new HttpClient();
+        private static ApiClient instance;
+
+        private ApiClient()
+        {
+            RunAsync().GetAwaiter().GetResult();
+        }
+
+        public static ApiClient Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    instance = new ApiClient();
+                }
+                return instance;
+            }
+        }
+
+        static async Task RunAsync()
+        {
+            client.BaseAddress = new Uri(Config.Host);
+            client.DefaultRequestHeaders.Accept.Clear();
+            client.DefaultRequestHeaders.Accept.Add(
+                new MediaTypeWithQualityHeaderValue("application/json"));
+            client.DefaultRequestHeaders.Add("Accept", "application/vnd.api+json; version=1");
+            try
+            {
+                await GetProjectAsync();
```

Comment:
Also sort of confused why this is called here.

Reply:
This is simply the structure being used at this point to test out that the client is connected correctly to Panoptes and can return JSON. It seems standard to have this info in a try/catch. Take a peek at the article linked on the other comment for more info. However, this will change somewhat structurally as I continue working on the client.

Reply:
Yeah, I guess I think it's weird that you call this code in the constructor for the class. But if it's just for testing I suppose that's fine.